lib/core/test/bench/db/Main.hs

```diff
@@ -224,8 +236,8 @@ main = withUtf8Encoding $ withLogging $ \trace -> do
 --
 -- Currently the DBLayer will only store a single checkpoint (no rollback), so
 -- the #Checkpoints axis is a bit meaningless.
-bgroupWriteUTxO :: DBLayerBench -> Benchmark
-bgroupWriteUTxO db = bgroup "UTxO (Write)"
+bgroupWriteUTxO :: (Int -> Int -> [TxOut]) -> String -> DBLayerBench -> Benchmark
```

Comment:
**Suggestion:** since the `(Int -> Int -> [TxOut])` type appears in quite a few type signatures (which are already quite long), perhaps define a type alias?

```hs
type MkTxOuts = Int -> Int -> [TxOut]
```

Reply:
I'm generally not a fan of type aliases for functions. I don't expect us to change this signature frequently. It would be a reasonable change to avoid too much refactoring work. But this is all in one module and fairly straight-forward.